plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -2072,6 +2072,8 @@ protected StartAnswer execute(StartCommand cmd) {
             //
             for (DiskTO vol : sortedDisks) {
                 if (vol.getType() == Volume.Type.ISO || deployAsIs && vol.getType() == Volume.Type.ROOT) {
+                    rootDiskTO = vol;
+                    resizeRootDiskOnVMStart(vmMo, rootDiskTO, hyperHost, context);
```

Comment:
@Pearl1594 is this required for ISO as well?

Reply:
Thanks @sureshanaparti It's not required for ISO , have made the necessary change.